.github/workflows/changelog.yml

```
@@ -0,0 +1,14 @@
+name: Check Changelog
```

Let's rename the workflow so that it is more meaningful when we see it on the list of checks for a pull request.

```suggestion
name: Changelog Updated
```